lib/features/base/base_controller.dart

```diff
@@ -66,4 +72,17 @@ abstract class BaseController extends GetxController with MessageDialogActionMix
       FpsManager().removeFpsCallback(fpsCallback!);
     }
   }
+
+  void injectAutoCompleteBindings(Session? session, AccountId? accountId) {
+    if (session == null || accountId == null) {
+      return;
+    }
```

Comment:
IMO. it is useless, because the `requireCapability` handle this

Reply:
I think it's necessary. Because in the case of `reload web`. At `reload` time `session` and `accountId` will be null, so it will continue with the `requireCapability ` statement and will be fired `exception` in `try/catch`. I don't want to waste time for the system to process when it is certain that `session` and `accountId` are null. WDYT? @hoangdat h

Reply:
okie, but for the case `session` and `accountId` not null, system need to spend time to check these conditions 2 times :). It up to you. If you keep that, please `log` before return. Thank